src/App/App.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import USAState from '../USAState/USAState';
+import ComparisonContainer from '../ComparisonContainer/ComparisonContainer';
+import Stat from '../Stat/Stat';
+import './App.scss';
+
+interface Props { 
+}
+
+interface State {
+  selectedUSAState: {};
+}
+
+class App extends Component<Props,State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      selectedUSAState: {}
+    }
+  }
+
+  render() {
```

Comment:
I really like the way we outlined the components here. This will make it easier to incorporate the data from the API on the next branch.